codemods/custom-property-replacer/README.md

```diff
@@ -0,0 +1,31 @@
+#### Custom Property Replacer
+
+The purpose of this PostCSS plugin is to replace deprecated custom properties with the new custom properties. To use this plugin, follow the following instructions.
+
+First, start by installing `postcss-cli`. You will use this to run the css transformation.
+
+`yarn add -D postcss-cli`
```

Comment:
I would maybe recommend to use `npx` to run the command, as it's usually a one time thing, so there is no need to have an extra dependency installed